### ISSUE TITLE
eventReceive/eventLeave is missing revert and relatedEvents

### ISSUE THREAD

**Issue by Voldemorten** [fullcalendar/fullcalendar#5610]
On `EventReceive` the `revert`-function and also the `relatedEvents`-object are missing. I don't know if the error is in the documentation or in the code. 

Codepen: https://codepen.io/MortenDL/pen/XWXBmRd

Output when logging the info element:
```
{
   "draggedEl":{

   },
   "event":{
      "title":"My Event 1",
      "start":"2020-06-29"
   },
   "view":{
      "type":"dayGridMonth",
      "dateEnv":{}
   }
}
```

**Comment by acerix** [comment]
Thanks, confirmed.

**Comment by arshaw** [comment]
same with eventLeave

**Comment by arshaw** [comment]
fixed in v5.2.0: https://codepen.io/arshaw/pen/VweoXNy?editors=0010
